event/oracle/register_oracle_event.go

```diff
@@ -67,7 +64,10 @@ func (e RegisterOracleEvent) verifyAndGetMsgApproveOracleRegistration(uniqueID,
 		return nil, fmt.Errorf("oracle's uniqueID does not match the requested uniqueID. expected(%s) got(%s)", approverUniqueID, uniqueID)
 	} else {
```

Comment:
It doesn't have to be  `if-else` statement.

Reply:
It meant we can possible `else` statement?

Reply:
I removed `else` condition.

Reply:
Yes. Because if-statement return error.

Reply:
OK. I got it.